### PR TITLE
Removed non working demos till vrogger

### PR DESCRIPTION
**Pull request by rabimba**
Removed all non-working/dead demo links till "vrogger" demo. Will clean up the rest and send another PR.

### CODE REVIEW

**Comment by cvan** [comment]
Forgive my ignorance; perhaps Kevin/Diego know. But what's vrogger?

**Comment by ngokevin** [comment]
It's a demo. He just looked through them and randomly stopped at that one to continue later.

File: README.md [hunk 1]
```diff
@@ -309,8 +294,6 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [SlideAmaze](http://slideamaze.com/) - Turns your slideshow presentation [into a 3D maze](http://fouryears.eu/2016/07/27/slide-a-maze/)
 - [A-Frame AR](https://github.com/autonome/aframe-ar) - Experimenting with AR using A-Frame and getUserMedia
 - [Outside Lands VR](https://ngokevin.github.io/aframe-magicavoxel-projects/outsidelands/) - VR preview of the San Francisco Outside Lands music festival
-- [Home/Hope](https://homehope.corriere.it/) )- Narrated journalism piece about humanitarian work in South Sudan. Featured on Italy’s top news site
```
Comment (ngokevin) [inline README.md]: I found this still works if we change https -> http

File: README.md [hunk 2]
```diff
@@ -296,7 +282,6 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Space Shark Room](http://liooil.neocities.org/VR_webStuff/room+07/room07.html) - Hinduist shark cage in space
 - [Metal Gear REX](http://www.jmvisualcreativity.es/vr_metal/) - Return to Shadow Moses. SNAKE!
 - [Meditation Orbs](http://www.skyislandsvr.com/pages/meditationorbs.html) - A place to meditate
-- [Welcome](http://drawvr.com/welcome/) - To A-Frame
```
Comment (ngokevin) [inline README.md]: This one still works for me

Comment (rabimba) [inline README.md]: Sorry that was a mistake! I thought I had seen that before and removed that as a duplicate. Will send another PR after I go through remaining list too. And correct this one and the one you noted above (https->http)

Comment (rabimba) [inline README.md]: Ok. you had already merged it. It(Line 299) was a duplicate of line 272

Comment (ngokevin) [inline README.md]: Oh, ok, never mind!
